src/machine/machine.c

```diff
@@ -323,8 +323,10 @@ int machine_load(Machine *m) {
                         int ifi;
 
                         r = extract_first_word(&p, &word, NULL, 0);
-                        if (r < 0)
-                                return log_error_errno(r, "Failed to parse NETIF: %s", netif);
+                        if (r == -ENOMEM)
+                                return log_oom();
+                        else
+                                log_warning_errno(r, "Failed to parse NETIF: %s", netif);
```

Comment:
No, this has two problems:

* `extract_first_word()` will return >1, if it has a new word, in which case you now spit out a bogus warning
* in case `extract_first_word()` returns <0, we are not allowed to use `word`.

You need to explicitly check for `r < 0` here and `continue`. Also, you can merge the `r == 0` below as `else if` branch.